pages/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="flex flex-col gap-5">
+    <div class="w-[300px] flex flex-row gap-3 items-center">
+      <div>Category</div>
```

Comment:
```suggestion
    <div class="flex items-center">
      <h2 class="mr-3">Category</h2>
```

Reply:
@archit-credilio I have resolved the comment